docs/chap-networking.md

```diff
@@ -214,6 +214,158 @@ The Polkadot Host must actively communicate with the network in order to partici
+:::definition
+
+An **authority identifier request** is a request that polkadot nodes can send to the authority discovery mechanism in order to request the `AuthorityId` of an authority node. The request has the following format:
+
+$$
+\texttt{get\_authority\_ids\_by\_peer\_id}{\left(\texttt{PeerId}\right)}
+$$
+
+**where**
+- $\texttt{PeerId}$ is the Polkadot node’s PeerId ([Definition -def-num-ref-](chap-networking#defn-peer-id)). 
+
+
+**expected response** 
+<!-- Since we may store the mapping across several sessions, a single. `PeerId` might correspond to multiple `AuthorityId`s -->
+The response to the previous query includes an enum with one of the following values:
+<!-- Option<HashSet<AuthorityDiscoveryId> -->
+
+| Value                  | Description                                            |
+|------------------------|--------------------------------------------------------|
+|  None                  | A type representing `no value`                           | 
+|  HashSet[authorityId]  | An unordered collection of unique `authorityId` elements | 
+
+
+**with**
+- `authorityId` the 256-bit identifier representing the public key of the requested `PeerId`. 
+
+:::
+
+
+<!-- AUTHORITY DISCOVER interacting with KADEMLIA -->
+#### -sec-num- Publishing local addresses {#sect-auth-discovery-worker-publishing}
+<!-- worker component: network.put_value( hash(authorityId) , SignedAuthorityRecord);  -->
+
+The authority discovery mechanism periodically sends `put_value` messages to the DHT in order to publish a `SignedAuthorityRecord` of the addresses of authorities it knows from its current authority sets. The `put_value` message to be sent to Kademlia DHT is created as follows.
+<!-- Retrieve authority identifiers of the current and next authority set.. -->
+
+###### Definition -def-num- Signed Authority Record {#defn-msg-signed-authority-record}
+:::definition
+
+The `SignedAuthorityRecord` is a Protobuf serialized structure representing the authority records and signature to send over the wire.
+It is defined in the following format:
+
+| Type               | Id  | Description                                           |
+|--------------------|-----|-------------------------------------------------------|
+| *AuthorityRecord*  | 1   | Serialized authority record                           |
+| `bytes`            | 2   | An Schnorrkel/Ristretto x25519 ("sr25519") signature  |
+| *PeerSignature*    | 3   | Serialized peer signature                             |
+
+**where**  
+
+**AuthorityRecord** is a serialized Protobuf structure that lists the addresses of authority nodes that are currently part of the authority set. 
+
+| Type             | Id | Description                                                           |
+|------------------|----|-----------------------------------------------------------------------|
+| `repeated bytes` | 1  | Possibly multiple `MultiAddr`ess through which a node can be connected | 
```

Comment:
```suggestion
| `repeated bytes` | 1  | Binary representation of zero or more multiaddresses through which a node is reachable | 
```

`MultiAddr` is probably how the Substrate struct is named, but the concept is just a "multiaddr".

I would mention specifically the binary representation of a multiaddress, because multiaddresses have both a string and binary representation.

Reply:
Agree.